modules/kuberay-operator/kuberay.tf

```diff
@@ -25,6 +25,7 @@ resource "helm_release" "kuberay-operator" {
 
 module "kuberay-workload-identity" {
   source              = "terraform-google-modules/kubernetes-engine/google//modules/workload-identity"
+  version             = "30.0.0"
```

Comment:
can you add a comment mentioning why we pin this?
can we also pin this in frontend and jupyter?

search for all refs: https://github.com/search?q=repo%3AGoogleCloudPlatform%2Fai-on-gke%20terraform-google-modules%2Fkubernetes-engine%2Fgoogle%2F%2Fmodules%2Fworkload-identity&type=code

Reply:
Fixed